odpf/optimus/runtime_service.proto

```diff
@@ -255,6 +255,20 @@ message JobSpecHook {
     repeated JobConfigItem config = 2;
 }
 
+message JobSpecResourceConfig {
+    string cpu = 1;
+    string memory = 2;
+}
+
+message JobSpecResource {
+    JobSpecResourceConfig request = 1;
+    JobSpecResourceConfig limit = 2;
+}
```

Comment:
Should we prefix them with `Metadata`? `Resource` is a term already used for something else in Optimus. It doesn't directly convey the message these are execution time resource limits/requests

Reply:
Oh right. Then, are `JobSpecMetadataResource` and `JobSpecMetadataResourceConfig` good enough?

Reply:
yeah sounds good.

Reply:
it's updated as suggested